.github/dependabot.yml

```diff
@@ -11,7 +11,15 @@ updates:
   - package-ecosystem: "pip"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
+    groups:
+      dev-patch-updates:
```

Comment:
What's the significance of "dev-" in the group names? My first thought was dev dependencies, but this appears to be for everything

Reply:
dep- works for me :+1: